core/src/G3Map.cxx

```diff
@@ -4,6 +4,201 @@
 #include <container_pybindings.h>
 #include <serialization.h>
 
+/* Special load/save for int64_t. */
```

Comment:
Maybe around mention that a similar encoding is also used in G3Vector.cxx?

These bit_count functions (incl. the one in g3Vector.cxx) could be templated and put in a header file pretty easily to reduce duplication.  Take advantage of bit_count([a, b, c]) = max([bit_count(a), bit_count(b), bit_count(c)]).

Reply:
Moved the storage functions to separate (private) header/source files, and used templates where appropriate.